web/rest/client/features/provider/contact/postContact.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Create contacts
+  In order to manage contacts
+  As a client admin
+  I need to be able to create them through the API.
+
+  @createSchema
+  Scenario: Create a contact
+    Given I add Company Authorization header
+     When I add "Content-Type" header equal to "application/json"
+      And I add "Accept" header equal to "application/json"
+      And I send a "POST" request to "/contacts" with body:
+    """
+      {
+          "name": "New",
+          "lastname": "Contact",
+          "email": "newcontact@example.com",
+          "workPhoneCountry": 68,
+          "workPhone": "445223"
```

Comment:
mobilePhone and otherPhone?

Reply:
Updated test to create a Contact with the 3 phones filled.